tests/upsAssets.spec.ts

```diff
@@ -0,0 +1,188 @@
+            responseTxt.slice(0, 1000)
+          );
+
+          switch (assetsDataObj!.name) {
+            case "atlas about page":
+              test.fixme(
+                expectedLanguage !== "en",
+                "Implement https://kontur.fibery.io/Tasks/Task/add-translated-About-pages-to-user-profile-api-repo-18359 to activate this test"
+              );
+              linksAtAtlasAboutPage.forEach((link) =>
+                expect(responseTxt).toContain(link)
+              );
+              break;
+
+            case "oam about page":
+              test.fixme(
+                expectedLanguage !== "en",
+                "Implement https://kontur.fibery.io/Tasks/Task/add-translated-About-pages-to-user-profile-api-repo-18359 to activate this test"
+              );
+              linksAtOAMAboutPage.forEach((link) =>
+                expect(responseTxt).toContain(link)
+              );
+              break;
+
+            case "disaster-ninja about page":
+              test.fixme(
+                expectedLanguage === "uk",
+                "Fix https://kontur.fibery.io/Tasks/Task/About-page-UK-locale-fix-go-to-map-link-and-github-link-19533 to activate this test"
+              );
+              linksAtDNAboutPage.forEach((link) =>
+                expect(responseTxt).toContain(link)
+              );
+              break;
+
+            case "smart-city about page":
+              test.fixme(
+                expectedLanguage !== "en",
+                "Implement https://kontur.fibery.io/Tasks/Task/add-translated-About-pages-to-user-profile-api-repo-18359 to activate this test"
+              );
+              linksAtSmartCityAboutPage.forEach((link) =>
+                expect(responseTxt).toContain(link)
+              );
+              break;
+
+            default:
+              // Handle terms and privacy pages with separate fixme and link checks
+              if (assetsDataObj!.name.includes("terms page")) {
+                test.fixme(
+                  expectedLanguage !== "en",
+                  "Implement https://kontur.fibery.io/Tasks/Task/add-translated-About-pages-to-user-profile-api-repo-18359 to activate this test"
+                );
+                linksAtTermsPage.forEach((link) =>
+                  expect(responseTxt).toContain(link)
+                );
+                polishTextAtTermsPage.forEach((text) =>
+                  expect(responseTxt).toContain(text)
+                );
+              }
+              if (assetsDataObj!.name.includes("privacy page")) {
+                test.fixme(
+                  expectedLanguage !== "en",
+                  "Implement https://kontur.fibery.io/Tasks/Task/add-translated-About-pages-to-user-profile-api-repo-18359 to activate this test"
+                );
+                linksAtPrivacyPage.forEach((link) =>
+                  expect(responseTxt).toContain(link)
+                );
+              }
+              break;
+          }
```

Comment:
no I won't do it because 
1 - you don't save any more lines of code. it makes no sense to create more code to follow DRY principle. So you break KISS principle
2 - i'll need ion future to support tests and add more checks/fix smth